site/content/docs/5.2/forms/select.md

```diff
@@ -67,6 +67,13 @@ As is the `size` attribute:
 
 Add the `disabled` boolean attribute on a select to give it a grayed out appearance and remove pointer events.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This **disabled** `<select>` element should not be used with this appearance because it does no respect the Orange Design System specifications.
```

Comment:
Strongly against the presence of this callout. Users can use this disabled select, it's our job to provide the right rendering.

Reply:
Resolved ?